packages/datetime/test/dateInputTests.tsx

```diff
@@ -53,6 +53,16 @@ describe("<DateInput>", () => {
         assert.isTrue(wrapper.find(TimePicker).isEmpty());
     });
 
+    it("popoverProps are passed to Popover", () => {
+        const popoverWillOpen = sinon.spy();
+        const wrapper = mount(<DateInput popoverProps={{ inline: true, position: Position.TOP, popoverWillOpen }} />);
+        wrapper.find("input").simulate("focus");
+        const popover = wrapper.find(Popover);
+        assert.strictEqual(popover.prop("inline"), true);
+        assert.strictEqual(popover.prop("position"), Position.TOP);
+        assert.isTrue(popoverWillOpen.calledOnce);
+    });
```

Comment:
I'd also like to see some tests asserting which props are not overridable. See https://github.com/palantir/blueprint/pull/940/files, e.g.